pages/osx/sed.md

```diff
@@ -12,7 +12,7 @@
 
 - Replace all occurrences of a string in a file, overwriting the file (i.e. in-place):
 
-`sed -i '' 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed -i 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
I'm not sure this is correct. From the `sed` manual:
```
-i[SUFFIX], --in-place[=SUFFIX]

              edit files in place (makes backup if SUFFIX supplied)
```

...so the `''` here is specifying a suffix of nothing to overwrite the file.

If anything, we would perhaps want the long-form of the option, like this:

```suggestion
`sed --in-place='' 's/{{find}}/{{replace}}/g' {{filename}}`
```